src/Winium.Desktop.Driver/CommandExecutors/NewSessionExecutor.cs

```diff
@@ -32,6 +31,17 @@ protected override string DoImpl()
             // Gives sometime to load visuals (needed only in case of slow emulation)
             Thread.Sleep(this.Automator.ActualCapabilities.LaunchDelay);
 
+            // Update running application process in Application class if it's exited base on input process name
+            if (this.Automator.Application.HasExited())
```

Comment:
@quangnt I think we should generalize this a bit, instead of `processname` we should use `realProcessname`

The term `realProcessname` indicate it's the real one not the launcher. 

And the comment `// Update running application process in Application class if it's exited base on input process name` --> `// Update running application process pointing to the real process instead of the launcher`

As a result, I process the code change to:
```
// Update running application process pointing to the real process instead of the launcher
ifif (this.Automator.Application.HasExited() && this.ExecutedCommand.Parameters["desiredCapabilities"]["processname"]){
    var processName = this.ExecutedCommand.Parameters["desiredCapabilities"]["processname"].ToString()
    this.Automator.Application.UpdateProcessBy(processName );
}
```